avalon/style/style.qss

```diff
@@ -4,6 +4,11 @@
     font-family: "Open Sans";
 }
 
+/*
+@ Selection highlight: #40576D
```

Comment:
This was like a quick access for testing color, since `qss` doesn't support declaring variables. Could be removed if you want it more cleaner.